plaso/parsers/locate.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+"""Parser for locate/updatedb database files."""
+
+import os
+
+from struct import unpack as struct_unpack
+from struct import error as struct_error
+
+from dfdatetime import posix_time
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.lib import errors
+from plaso.lib import specification
+from plaso.parsers import interface
+from plaso.parsers import manager
+
+
+class LocateDatabaseEvent(events.EventData):
+  """Linux Locate Database event data.
+
+  Attributes:
+    folder_path (str): full folder path.
+  """
+
+  DATA_TYPE = 'linux:locate'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(LocateDatabaseEvent, self).__init__(data_type=self.DATA_TYPE)
+    self.folder_path = None
+
+
+class LocateDatabaseFile(object):
```

Comment:
Please use dtFabric for an example see https://github.com/log2timeline/plaso/blob/main/plaso/parsers/winjob.py

Reply:
Using dtFabric now